### ISSUE TITLE
internal server error on `/post/comment`

### ISSUE THREAD

**Issue by AaditT** [chemokita13/beReal-api#17]
Hey, I am trying to hit the `/post/comment` endpoint with valid token, postId, and comment. I keep getting 500 internal server error.

**Comment by chemokita13** [comment]
Het, that seems like Bereal have changed the endpoint, i will work on it!

**Comment by basti394** [comment]
Is this issue fixed already?

**Comment by chemokita13** [comment]
Ey! @basti394 I think its already solved! You can try it!

**Comment by basti394** [comment]
@chemokita13 thanks for the response, but unfortunately I still get a 500 exception. As a request Body I have my userId, the post Id and the message.

**Comment by chemokita13** [comment]
> @chemokita13 thanks for the response, but unfortunately I still get a 500 exception. As a request Body I have my userId, the post Id and the message. 

Are you correct sending the params and the login token? Could you share a screenshot for seeing how do you make the request?

**Comment by basti394** [comment]
![Screenshot 2024-01-24 at 20 10 01](https://github.com/chemokita13/beReal-api/assets/76596010/7a92483c-4a40-494b-9ef3-1e342f419bcd)

The token should correctly be attached, because every other endpoint works.

**Comment by chemokita13** [comment]
Are you sending your own userId or the user's post id?

**Comment by basti394** [comment]
I send my own user Id. Should I send the other one?

**Comment by chemokita13** [comment]
Yes, I mean, if I want to comment on my @friend post, I have to send my @friend userId and the postId of my @friend.
Also If you wanna comment in your own post the userId will be your userId

**Comment by basti394** [comment]
Ahhh, okay thanks 👍 
Btw I guess it would be better to then also return another code than 500, something more like 400

**Comment by chemokita13** [comment]
Yeah, it's pending to change and improve all the server responses but its too large and idk when I will be able to solve it hahaha
